src/sentry/snuba/metrics/extraction.py

```diff
@@ -37,6 +37,10 @@
 
 logger = logging.getLogger(__name__)
 
+NEXT_SPEC_FEAT_FLAG = "organizations:on-demand-metrics-query-spec-version-two"
```

Comment:
TODO: We need to create this Flagr feature flag.

Reply:
Flag#558